snapcraft/main.py

```diff
@@ -81,25 +86,27 @@
     'upload',
 ]
 
-try:
-    version = pkg_resources.require('snapcraft')[0].version
-except pkg_resources.DistributionNotFound:
-    version = 'devel'
+
+def _version():
```

Comment:
but wait 'getters' in python were just the name, same for 'setters'; wouldn't this be a property in some sense?